go/vt/vttablet/tabletserver/vstreamer/engine.go

```diff
@@ -383,6 +385,10 @@ func (vse *Engine) setWatch() {
 	})
 }
 
+func (vse *Engine) getStreamersActive() int64 {
+	return int64(len(vse.streamers))
+}
```

Comment:
Slice length is not concurrency safe so we should use the mutex. Also, IMO a better name:
```
func (vse *Engine) getVStreamerCount() int64 {
	vse.mu.Lock()
	defer vse.mu.Unlock()
 	return int64(len(vse.streamers))
 }
```

Reply:
@mattlord sounds good 👍

I might be worrying about introducing a lock too much but could I hypothetically use `atomic.AddUint64()`/`atomic.LoadUint64()` instead and incr/decr it from `.Stream()` instead and is there a preference?

Reply:
I don't think that's needed here, because the engine doesn't change THAT often (not typically starting/ending hundreds or even thousands of vstreams on a tablet in a second, or even minute). @rohit-nayak-ps, what do you think?